### PR TITLE
5 about page

### PR DESCRIPTION
**Pull request by SmruthyJoseph**
**About page** created.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1887,13 +1887,13 @@ ul.timeline > li .timeline-cnt ul li a{
...
+
     .posts-outer{
         margin-top: 30px;
     }
     .posts-outer .top-post{
         margin-bottom: 110px;
-        background: url(../images/news-banner-bg.png) repeat-x left 35% / 100%;
+        background: url(assets/images/news-banner-bg.png) repeat-x left 35% / 100%;
     }
     .posts-outer .top-post .left{
```
Comment (AbhijithHaridas) [inline style.css]: Only these few style changes for the entire about page?

**Review by AbhijithHaridas — COMMENTED** [review]
@SmruthyJoseph is this completed fully?

File: style.css (hunk 1 above)
Comment (SmruthyJoseph) [reply style.css]: No, I will change it

**Comment by AbhijithHaridas** [comment]
### Disqualified

**Pull request closed without merging**